.github/workflows/ci.yml

```diff
@@ -332,10 +335,7 @@ jobs:
           if [ -f eval-summary.json ]; then
             combination_score=$(jq '.categories.combination' eval-summary.json)
             echo "Combination category score: $combination_score%"
-            if (( $(echo "$combination_score < 85" | bc -l) )); then
```

Comment:
let's stop failing these and let's find a better spot to call passing

Reply:
what does "find a better spot to call passing" mean?

Reply:
id rather take 4o-mini out of CI eval models than completely remove combos from CI